config/initializers/devise.rb

```diff
@@ -23,6 +23,8 @@
 
   config.sign_out_via = :delete
 
+  config.timeout_in = 30.minutes
```

Comment:
Is it 30 minutes after the session start? 
or 30 minutes of inactivity?

In the first case, 30 minutes might not be enough for candidates to complete the application.

Reply:
It's the latter I believe, it's based on inactivity: https://www.rubydoc.info/github/plataformatec/devise/master/Devise/Models/Timeoutable

Reply:
Since it is inactivity do we still want the timeout to be 7.days?

Reply:
Yes, I think it's still reasonable 👍

Reply:
ok, that's an easy change, let's use 7.days for now